core/tests/puppeteer-acceptance-tests/spec/exploration-editor-tests/savedraft-publish-and-discard-the-changes.spec.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
```

Comment:
Filename should be `save-draft-publish-and-...`

Reply:
@StephenYu2018  savedraft is a single word here.
![image](https://github.com/oppia/oppia/assets/136263179/99c72699-e6b7-493c-8af4-4bc7a50d3e79)

If still need to change the name, PTAL. /cc @seanlip

Reply:
done